exporters/otlp/internal/otlpconfig/envconfig.go

```diff
@@ -145,6 +145,18 @@ func (e *EnvOptionsReader) GetOptionsFromEnv() []GenericOption {
 	return opts
 }
 
+func endpointOptions(endpoint string, endpointOption func(endpoint string) GenericOption, insecureOption func(insecure bool) GenericOption) []GenericOption {
+	var endpointOptions []GenericOption
+
+	endpointOptions = append(endpointOptions, insecureOption(strings.HasPrefix(endpoint, "http://")))
+
+	endpoint = strings.Replace(endpoint, "https://", "", 1)
+	endpoint = strings.Replace(endpoint, "http://", "", 1)
```

Comment:
This is very strange to me. gRPC is working over HTTP. we should not remove the scheme. Even if magically it magically works I would consider it as a hack. Was it integration tested?

Reply:
Alternatively we can use functions like `strings.ToLower` and `strings.TrimSpace` to probably solve most of the issues.

Reply:
I refactored it towards the solution with `ToLower` & `TrimSpace`

Reply:
This comment can be resolved